### PR TITLE
[new release] rope (0.6.3)

### PR DESCRIPTION
**Pull request by Chris00**
Ropes (heavyweight strings)

- Project page: <a href="https://github.com/Chris00/ocaml-rope">https://github.com/Chris00/ocaml-rope</a>
- Documentation: <a href="https://Chris00.github.io/ocaml-rope/doc">https://Chris00.github.io/ocaml-rope/doc</a>

##### CHANGES:

- Compatibility with OCaml 5.

### CODE REVIEW

File: packages/rope/rope.0.6.3/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler" ]
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
```
Comment (mseri) [inline packages/rope/rope.0.6.3/opam:4]:
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```

File: packages/rope/rope.0.6.3/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+homepage: "https://github.com/Chris00/ocaml-rope"
+dev-repo: "git+https://github.com/Chris00/ocaml-rope.git"
+bug-reports: "https://github.com/Chris00/ocaml-rope/issues"
+doc: "https://Chris00.github.io/ocaml-rope/doc"
+tags: [ "datastructure" ]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/rope/rope.0.6.3/opam:11]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/rope/rope.0.6.3/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "5.00.0"}
```
Comment (mseri) [inline packages/rope/rope.0.6.3/opam:16]:
```suggestion
  "ocaml" {>= "5.0"}
```

**Comment by raphael-proust** [comment]
@Chris00 Can you upstream the changes made by @mseri to avoid having to do them again on the next release?

**Comment by Chris00** [comment]
Thanks @mseri !

**Pull request merged**